forest/src/cli/config.rs

```diff
@@ -12,6 +12,9 @@ pub struct Config {
     pub data_dir: String,
     pub genesis_file: Option<String>,
     pub drand_dist_public: DistPublic,
+    pub enable_rpc: bool,
+    pub rpc_port: Option<String>,
+    pub rpc_listen: Option<String>,
```

Comment:
These make more sense to be flags on the daemon command (although can still keep this) anyone else have thoughts on this?